frontend/sw360-portlet/src/main/webapp/html/components/view.jsp

```diff
@@ -342,14 +289,24 @@
             }
 
             function renderComponentNameLink(name, type, row) {
-                return renderLinkTo(makeComponentFriendlyUrl(row.id, '<%=PortalConstants.PAGENAME_DETAIL%>'), name);
+                return renderLinkTo(replaceFriendlyUrlParameter('<%=friendlyComponentURL%>', row.id, '<%=PortalConstants.PAGENAME_DETAIL%>'), name);
             }
 
-            function makeComponentFriendlyUrl(componentId, page) {
-                var portletURL = '<%=friendlyComponentURL%>'
+            function renderLicenseLink(lics, type, row) {
+                var licensePortletURL = '<%=friendlyLicenseURL%>'
+                    .replace("components", "licenses");// DIRTY WORKAROUND
```

Comment:
yes, dirty! Might even replace to much?

Reply:
Basically we need a better way to generate a porlet URL for licenses.
I changed it to `.replace(/components/g, "licenses");` this will only replace the first match.